src/pybind/mgr/cephadm/module.py

```diff
@@ -1566,30 +1566,37 @@ def blink_device_light(self, ident_fault: str, on: bool, locs: List[orchestrator
 
         If you must, you can customize this via::
 
-          ceph config-key set mgr/cephadm/lsmcli_blink_lights_cmd '<my jinja2 template>'
+          ceph config-key set mgr/cephadm/blink_device_light_cmd '<my jinja2 template>'
+          ceph config-key set mgr/cephadm/<host>/blink_device_light_cmd '<my jinja2 template>'
 
-        See templates/lsmcli_blink_lights_cmd.j2
+        See templates/blink_device_light_cmd.j2
         """
         @forall_hosts
         def blink(host, dev, path):
-            j2_ctx = {
+            tpl_name = 'blink_device_light_cmd.j2'
+
+            # Check if there is a custom template per host. Global
+            # templates are handled by the TemplateMgr::render()
+            # method automatically.
+            tpl_name_per_host = '{}/blink_device_light_cmd'.format(host)
+            if self.get_store(tpl_name_per_host, None) is not None:
+                tpl_name = tpl_name_per_host
+
+            cmd_line = self.template.render(tpl_name, {
                 'on': on,
                 'ident_fault': ident_fault,
                 'dev': dev,
                 'path': path
-            }
-
-            lsmcli_blink_lights_cmd = self.template.render('lsmcli_blink_lights_cmd.j2', j2_ctx)
```

Comment:
render already allows you to do that aready. can you add this to render instead? 

https://github.com/ceph/ceph/blob/8a2f2d82f49df1d5a94e97328f8b04a783d159d5/src/pybind/mgr/cephadm/template.py#L89-L94

Reply:
Adding the host as search parameter to the key to access the key-value store has been relocated to the template render method.